client/src/components/Attachment/UploadAttachment.js

```diff
@@ -25,26 +26,30 @@ const GQL_UPDATE_ATTACHMENT = gql`
   }
 `
 
-const UploadAttachment = ({ getRelatedObject, edit, saveRelatedObject }) => {
+const UploadAttachment = ({
+  edit,
+  relatedObjectType,
+  relatedObjectUuid,
+  saveRelatedObject
+}) => {
   const [error, setError] = useState(null)
   const [remove, setRemove] = useState(false)
   const [uploadedList, setUploadedList] = useState([])
-  const relatedObject = getRelatedObject()
 
-  const attachmentSave = async e => {
+  const attachmentSave = async(e, uuid) => {
     const file = e.target.files[0]
```

Comment:
These changes are not related to this PR but there is a bug here, if the selected file is empty it will crash.

Reply:
This error occurs when you open the file selector to upload an attachment and then close it without selecting a file after it was opened. It doesn't occur the first time you want to make an upload. @gjvoosten